cmd/chains.go

```diff
@@ -58,20 +58,17 @@ func filesAdd(ctx *config.Context, dir string) error {
 		}
 		byt, err := os.ReadFile(pth)
 		if err != nil {
-			fmt.Printf("failed to read file %s, skipping...\n", pth)
-			continue
+			return fmt.Errorf("failed to read file %s, error: %v", pth, err)
 		}
 		var c core.ChainProverConfig
 		if err := json.Unmarshal(byt, &c); err != nil {
-			fmt.Printf("failed to unmarshal file %s, skipping...\n", pth)
-			continue
+			return fmt.Errorf("failed to unmarshal file %s, error: %v", pth, err)
 		}
 		if err := c.Init(ctx.Codec); err != nil {
 			return err
 		}
 		if err = ctx.Config.AddChain(ctx.Codec, c); err != nil {
-			fmt.Printf("%s: %s\n", pth, err.Error())
-			continue
+			return fmt.Errorf("failed add chain %s, error: %v", pth, err)
```

Comment:
`failed to add chain ...` is correct?

Also, could you add an error msg for `c.Init()` at line 68? @dongrie

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/105/commits/e4c140f59be72296ec2b408165ea317e0746d573
Fixed error message and add error msg to c.Init()